lib/dialects/mysql/query-generator.js

```diff
@@ -151,6 +153,64 @@ const QueryGenerator = {
     return `ALTER TABLE ${this.quoteTable(tableName)} CHANGE ${attrString.join(', ')};`;
   },
 
+  handleSequelizeMethod(smth, tableName, factory, options, prepend) {
```

Comment:
I added code to handle MySQL JSON queries. It's pretty similar to the sqlite query-generator.